config/samples/mellanox.com_v1alpha1_macvlannetwork.yaml

```diff
@@ -1,7 +1,31 @@
+# Copyright 2021 NVIDIA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 apiVersion: mellanox.com/v1alpha1
 kind: MacvlanNetwork
 metadata:
-  name: macvlannetwork-sample
+  name: example-macvlannetwork
 spec:
-  # Add fields here
-  foo: bar
+  networkNamespace: "default"
+  master: "ens2f0"
+  mode: "bridge"
+  mtu: 1500
+  ipam: |
+    {
+      "type": "whereabouts",
+      "range": "192.168.2.225/28",
```

Comment:
can we change the example to be /24 ?

currently this gives you 11 addresses (with the exclude list below)

Reply:
Done